crates/trigger-http2/src/lib.rs

```diff
@@ -67,7 +67,8 @@ pub(crate) type InstanceState = ();
 
 /// The Spin HTTP trigger.
 pub struct HttpTrigger {
-    listen_addr: SocketAddr,
+    /// The address the server will listen on.
+    addr_to_bind: SocketAddr,
```

Comment:
I wouldn't change this; the CLI flag is `--listen`...

Reply:
I hear ya. I wanted to draw attention to the fact that this might not be the actual `SocketAddr` the socket is listening on (in the case of a 0 port), but perhaps that's not worth the potential confusion in the other direction.

Reply:
By the way, the CLI arg is actually `--address`... But I'll still change it back.

Reply:
Unfortunately it is a trick:

```rust
#[clap(long = "listen", ...)]
pub address: SocketAddr,
```